extension.js

```diff
@@ -128,6 +129,11 @@ class ReferenceProvider {
         if (!newUriPath.match(/\.js$/i)) newUriPath += '.js';
 
         const newUri = vscode.Uri.file(newUriPath);
+
+        if (!fs.existsSync(newUri.fsPath)) {
```

Comment:
If its not critical I completely agree. Since it's reported and seemed like an easy fix I thought lets apply the change.